resources/authgear/templates/en/web/settings_close.html

```diff
@@ -0,0 +1,38 @@
+{{ template "__page_frame.html" . }}
+
+{{ define "page-content" }}
+<div class="pane twc-container-vertical pt-8 pb-5 px-6 tablet:px-8 desktop:px-8">
+
+<h1 class="primary-txt m-0 text-center text-xl font-bold">{{ template "settings-close-page-title" }}</h1>
+
+<p class="text-sm break-words primary-txt m-0 text-center">{{ template "settings-close-page-description" }}</p>
+
+<button
+  id="return-button"
+  class="hidden"
+  type="button"
+  data-action="click->original-referrer#returnToReferrer"
+></button>
+
+{{ template "__watermark.html" . }}
+</div>
+
+<script nonce="{{ $.CSPNonce }}">
+(function() {
+  var authgear = window.authgear || {};
+  authgear.close = function() {
+    const hasParent = window.parent !== window;
+    if (!hasParent) {
+      // Remove close page from history
+      window.history.replaceState(null, '', window.location.href);
+      location.href = document.referrer;
+      return;
+    }
+
+    window.close();
```

Comment:
If the Web SDK supports two modes: 1) open in the same tab, 2) open in a new tab, then we should

1. If open in the same tab, then we should call `history.go(-n)` where `n` is the correct number that brings the end-user back to the app.
2. If open in the new tab, then the redirectURI is supposed to be `javascript:window.close()`. And if it is `href` of a `<a>`, then visiting the link will close the tab.